### DISCUSSION TITLE
How do I disable storybook crash reporting?

### DISCUSSION THREAD

**Discussion by mikelehen** [storybookjs/storybook#19687] [Help] [answered]
I had an issue in my code and Storybook logged the details and asked:

> Would you like to send crash reports to Storybook?

I accidentally said yes, but I don't actually want storybook sending crash reports all the time.  I'd like to disable it, but I can't figure out how.  Where is this configuration persisted?

**Comment by jonniebigodes** [comment]
@mikelehen, thanks for reaching out to us with your issue. We appreciate it. If you wouldn't mind checking the [telemetry documentation](https://storybook.js.org/docs/react/configure/telemetry#crash-reports-disabled-by-default), specifically the crash reports section. It was information on how to do it.

Have a great weekend.

Stay safe

**Reply by mikelehen** [reply]
@jonniebigodes Thanks for the reply! I had actually looked at those docs and checked the .storybook/main.ts in my project and it does not have `enableCrashReports: true` in it.  In fact no changes were made to any files in my project as far as I can tell (`git status` doesn't show any changes).

I dug into the source code and solved this for myself (not sure why I didn't do that from the start). I posted a new answer myself.  Appreciate your reply regardless!

**Reply by jonniebigodes** [reply]
No need to thank whatsoever. Glad to at least provide you with some context on the issue.

**Comment by mikelehen** [comment, marked as answer]
Actually, I checked the source code and it was pretty easy to figure out.  You need to delete `node_modules/.cache/storybook` in your project. After deleting that, it stopped auto-sending them and I again started getting the "Would you like to send crash reports to Storybook?" prompt again.

For reference, this finding is based on:
* The setting is persisted to cache here: https://github.com/storybookjs/storybook/blob/65957e6789bc57d506ca63e6f3be3927f110565e/code/lib/core-server/src/withTelemetry.ts#L24
* The cache is created here: https://github.com/storybookjs/storybook/blob/65957e6789bc57d506ca63e6f3be3927f110565e/code/lib/core-common/src/utils/cache.ts#L4
* The directory is resolved here: https://github.com/storybookjs/storybook/blob/65957e6789bc57d506ca63e6f3be3927f110565e/code/lib/core-common/src/utils/resolve-path-in-sb-cache.ts#L21

Note that if you're not in a project it will apparently be cached in `.cache/storybook` of your current directory: https://github.com/storybookjs/storybook/blob/65957e6789bc57d506ca63e6f3be3927f110565e/code/lib/core-common/src/utils/resolve-path-in-sb-cache.ts#L19
